### PR TITLE
[shopsys] Removed trailing whitespaces from markdown files

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| #457
|New feature| Yes 
|BC breaks| No 
|Fixes issues| #457
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by boris-brtan** [comment]
rebased on actual master

File: UPGRADE.md [hunk 1]
```diff
@@ -67,6 +67,20 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - if you're using Docker-sync, add the directories mentioned in the PR into `sync_exclude` section of your `docker-sync.yml` to make the synchronization faster as well
 - *(optional)* [#557 - php-fpm image has standard workdir (/var/www/html) in ci stage](https://github.com/shopsys/shopsys/pull/557)
     - update your `docker/php-fpm/Dockerfile` and `kubernetes/deployments/webserver-php-fpm.yml` according to [the pull request](https://github.com/shopsys/shopsys/pull/557) to simplify the CI build
+- [#580 Removed trailing whitespaces from markdown files ](https://github.com/shopsys/shopsys/pull/580)
```
Comment (simara-svatopluk) [inline UPGRADE.md]: @TomasLudvik @boris-brtan 
There is a whitespace in `files ]` could you remove it?
This should be definitely `optional`, if it isn't then it doesn't only remove trailing whitespaces

File: UPGRADE.md [hunk 2]
```diff
@@ -128,6 +142,22 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - modify your `routing_front.yml`, change configuration for routes `front_image`, `front_image_without_type` - replace `/content/` by `/%shopsys.content_dir_name%/`
 - *(optional)* [#535 added .dockerignore files](https://github.com/shopsys/shopsys/pull/535)
     - if you're using Docker-sync, add the directories mentioned in the PR into `sync_exclude` section of your `docker-sync.yml` to make the synchronization faster
+- [#580 Removed trailing whitespaces from markdown files ](https://github.com/shopsys/shopsys/pull/580)
+    - remove these lines from `.dockerignore`
```
Comment (PetrHeinz) [inline UPGRADE.md]: `.dockerignore` is in version control, there is no need to modify it when upgrading the monorepo

File: UPGRADE.md [hunk 3]
```diff
@@ -128,6 +142,22 @@ There is a list of all the repositories maintained by monorepo, changes in log b
...
+        project-base/*.md
+        project-base/docs
+        ```
+    - run `docker-compose down` to turn off your containers
+    - *(MacOS, Windows only)*
+        - run `docker-sync clean` so your volumes will be removed
+        - remove excluding of `docs` folder from `docker-sync.yml`
+        - run `docker-sync start` to create volumes
+    - run `docker-compose up -d --build --force-recreate` to start application
+    - phing target for checking and fixing standards has changed, update `build.xml` according to the changes
```
Comment (PetrHeinz) [inline UPGRADE.md]: `build.xml` is in version control, there is no need to modify it when upgrading the monorepo

**Comment by boris-brtan** [comment]
hi @simara-svatopluk, @PetrHeinz  sorry for the late response but this is hard to find. should i create pr for the fixes or issue? or is it better to wait for the release process?

**Comment by simara-svatopluk** [comment]
@boris-brtan if it is just a fix, then I would prefer short consultation with colleague, PR, CR+tests by another colleague, BV and merge. And creating an issue in Yodiz and on our table.

**Pull request merged**